torchgeo/datasets/l8biome.py

```diff
@@ -0,0 +1,260 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""L8 Biome dataset."""
+
+import glob
+import os
+from typing import Any, Callable, Dict, List, Optional, Sequence, cast
+
+import matplotlib.pyplot as plt
+import torch
+from rasterio.crs import CRS
+from torch import Tensor
+
+from .geo import RasterDataset
+from .utils import BoundingBox, download_url, extract_archive
+
+
+class L8Biome(RasterDataset):
+    """L8 Biome datasets.
+
+    The `L8 Biome <https://landsat.usgs.gov/landsat-8-cloud-cover-assessment-validation-data>`__ dataset # noqa: E501
```

Comment:
```suggestion
    The `L8 Biome <https://landsat.usgs.gov/landsat-8-cloud-cover-assessment-validation-data>`__ dataset
```
This is included in the documentation. Comments inside of docstrings aren't comments.

Reply:
This was to combat the warning generated by flake8 (E501 line too long (104 > 88 characters)). Should we still remove it?

Reply:
You should add it to the closing `"""` line. You can verify that it is currently being erroneously included in the docs by viewing https://torchgeo--1200.org.readthedocs.build/en/1200/api/datasets.html#l8biome.